src/cli/utils/fileSystem.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from "fs";
+import { FileStatus } from "../../types/exercises.types";
+
+/**
+ * Returns status of file with name starting with `partialFileName` in `path`
+ * @param path Path to look into
+ * @param partialFileName Partial name to match
+ */
+export const isFilePresent = (
+    path: string,
+    partialFileName: string,
+): FileStatus => {
+    try {
+        const files = fs
+            .readdirSync(path)
+            .map((file) => file.startsWith(partialFileName));
+        if (files.length > 0) {
+            if (files.length === 1) {
+                return FileStatus.SUCCESS;
+            }
+            return FileStatus.INCONCLUSIVE;
+        }
+        // Should never reach here
+        return FileStatus.FAILURE;
+    } catch (_) {
+        return FileStatus.FAILURE;
+    }
+};
+
+/**
+ * A utility function to list all the files present inside a folder
```

Comment:
```suggestion
 * A utility function to list all the file names present inside a folder
```
bc you returned strings rather than file node

Reply:
Good catch!